virtinst/virtinstall.py

```diff
@@ -1231,7 +1284,7 @@ def main(conn=None):
         if xml:
             print_stdout(xml, do_force=True)
     else:
-        start_install(guest, installer, options)
+        start_install(conn, guest, installer, options)
```

Comment:
Small comment, all this conn passing isn't needed, you can grab conn from `guest.conn`